server/repositories/offender.js

```diff
@@ -75,17 +80,17 @@ function offenderRepository(prisonApiHttpClient, incentivesApiHttpClient) {
   }
 
   function getAdjudicationsFor(offenderNo) {
-    return prisonApiHttpClient.get(
-      `${prisonApiBaseUrl}/offenders/${offenderNo}/adjudications`,
+    return adjudicationsApiHttpClient.get(
+      `${adjudicationsApiBaseUrl}/adjudications/${offenderNo}/adjudications`,
       {
-        'Page-Limit': config.prisonApi.adjudications.pageLimit,
+        'Page-Limit': config.adjudicationsApi.adjudications.pageLimit,
       },
     );
   }
 
   function getAdjudicationFor(offenderNo, adjudicationNo) {
-    return prisonApiHttpClient.get(
-      `${prisonApiBaseUrl}/offenders/${offenderNo}/adjudications/${adjudicationNo}`,
+    return adjudicationsApiHttpClient.get(
+      `${adjudicationsApiBaseUrl}/adjudications/${offenderNo}/charge/${adjudicationNo}`,
```

Comment:
[Here is this endpoint](https://github.com/ministryofjustice/hmpps-manage-adjudications-api/blob/9e4fb75fe356845eda7c96c66aab998f792aecee/src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/controllers/AdjudicationSummaryController.kt#L38-L64) in the AdjudicationsAPI